target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -635,6 +636,11 @@ ramips_setup_macs()
 		wan_mac=$(mtd_get_mac_binary factory 4)
 		lan_mac=$(macaddr_add "$wan_mac" 1)
 		;;
+	netgear,ex3700|\
+	netgear,ex6130)
+		wan_mac=$(mtd_get_mac_binary factory 0x2e)
+		lan_mac=$(mtd_get_mac_binary factory 0x28)
```

Comment:
You have a one-port. Why do you set this?

Reply:
only lan mac is used.
wan mac may be used if you want to use the device as real router.

Would be unused else...

Reply:
But have you checked that the vendor actually provides a MAC address at both locations?
To me it looks like a waste of addresses to provide four addresses for a device that only has three interfaces, so I wouldn't expect the vendor to do so. However, if you actually have valid addresses in both 0x28 and 0x2e, then you should indeed provide them here.

However, note that you already have `mtd-mac-address = <&factory 0x28>;` for &ethernet, so you can drop the lan_mac line in any case.

Reply:
From one of my previous posts:

> WiFi Analyzer and ARP pointed out that original firmware uses 9B for eth and 2.4G wifi, 9A for 5G Wifi. 

I will push last change soon-ish

Reply:
Oh, I'm sorry, you already posted a detailed description above and we already discussed it.
I'm jumping too much between different PRs...
So, just drop lan_mac here and keep wan_mac as it is.

Please rebase onto current master, as we have changed 02_network to contain those values in hexadecimal notation. You can just add your devices to the block
```
	8devices,carambola|\
	alfa-network,w502u|\
	arcwireless,freestation5|\
	netgear,wnce2001)
```
then.